doc/changes.md

```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
+* Add development toctree and [PyGMT team page](https://www.pygmt.org/latest/team.html) ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
```

Comment:
I feel people may not understand what "development toctree" means. Perhaps we can just say:

> Add a "PyGMT Team" page

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/5426bbef085c3de95b2401e457992230ecfe5aee